spec/std/file_spec.cr

```diff
@@ -648,6 +648,59 @@ describe "File" do
     end
   end
 
+  describe "append" do
+    it "can create a new file" do
+      filename = "#{__DIR__}/data/temp_append.txt"
```

Comment:
You should make sure this path really doesn't exist before writing to it.
And I don't think these files should go in the data dir, just use tmp.